book/installation.md

```diff
@@ -1,6 +1,46 @@
 ## Installation
 
-(still WIP, please contribute!):
+To test Reth, you will need to have Geth and Lighthouse installed. 
+
+#### Mac
+```bash
+brew install geth lighthouse
+```
+
+#### Ubuntu
+1. First, update the package list:
+```bash
+sudo apt-get update
+```
+
+
+2. Then, install Geth:
+```bash
+sudo apt-get install geth
+```
+
```

Comment:
As mentioned above in my comment, I don't think any of this should be here. This book is for end users, so they would never need geth. At some point they will need a CL client but we do not support any now, so really the only change we need here is saying in the readme that people need geth to run the tests.